sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2257,58 +2289,73 @@ def update_cluster_ips(
             exceptions.FetchIPError: if we failed to get the IPs. e.reason is
```

Comment:
nit: upd docstr (maybe search FetchIPError and replace them all`

Reply:
Good catch! Updated. Thanks!